dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -273,11 +273,19 @@ private static DependencyGraph levelOrder(
       DependencyNode dependencyNode = item.dependencyNode;
       DependencyPath forPath = new DependencyPath();
```

Comment:
`forPath` is a very strange name. Any ideas for making it clearer?

Reply:
Updated to path.